sdk/devcenter/azure-developer-devcenter/tests/testcase.py

```diff
@@ -14,10 +14,11 @@ class DevcenterTest(AzureRecordedTestCase):
     def create_client(self, endpoint):
         credential = self.get_credential(DevCenterClient)
         return DevCenterClient(endpoint=endpoint, credential=credential)
-    
+
+
 DevcenterPowerShellPreparer = functools.partial(
     PowerShellPreparer,
```

Comment:
Not going to block this PR on this, but we should update this later to use the EnvironmentVariableLoader